### ISSUE TITLE
failure for empty mtl tables

### ISSUE THREAD

**Issue by forero** [desihub/fiberassign#143]
Got a failure when the  input `mtl` file does not have any targets in a given tile

```
Traceback (most recent call last):
  File "/global/homes/f/forero/fiberassign/bin/fiberassign", line 293, in <module>
    fiber_data = add_fiber_data_columns(tile_data, fiber_data, mtl_data[mtl_tile_indices[int(sky_id)]], sky_data, positioner_data)
  File "/global/homes/f/forero/fiberassign/bin/fiberassign", line 266, in add_fiber_data_columns
    fiberassign_data = join(fiberassign_data, mtl_data[ii][columns], join_type='left', keys='TARGETID')
  File "/global/common/software/desi/cori/desiconda/20180709-1.2.6-spec/conda/lib/python3.6/site-packages/astropy/table/operations.py", line 123, in join
    uniq_col_name, table_names, col_name_map, metadata_conflicts)
  File "/global/common/software/desi/cori/desiconda/20180709-1.2.6-spec/conda/lib/python3.6/site-packages/astropy/table/operations.py", line 574, in _join
    raise ValueError('input tables for join must both have at least one row')
ValueError: input tables for join must both have at least one row
```

**Comment by rncahn** [comment]
Doesn't sound right.  I thought fiberassignment checked to see which tiles
had targets and which didn't.  Maybe there is a problem if there are
standard stars and or sky fibers, but no real targets.

**Comment by forero** [comment]
The failure is in the python script. I know how to solve it. I just left this here to keep track of it.

**Comment by forero** [comment]
Bug fixed in commit ed33a004802001b43a9c06130a38c497e057a625
